_bibliography/references.bib

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+---
+
+% ordered by year (descending) and then by alphabetical order
+
+# Publications
+
+## 2017
+
+@article{batut2017community,
```

Comment:
what about this file: https://github.com/usegalaxy-eu/website/blob/master/_bibliography/freiburg.bib#L666

Reply:
I thought we could need more references (not only ours) here, but also in some posts. It is why I created an extra file

Reply:
Even if we have duplicated entries then?

Reply:
I added more references there